templates/author/blogger_detail.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+
+{% block title %}Author{% endblock %}
+
+{% block content %}
+<h1 class="text-success">{{author.username}}</h1>
+<hr>
+<div>
+    <p>{{author.bio_detail}}</p>
+</div>
+<hr>
+<h3 class="text-success">All Blogs</h3>
+{% if auth_posts %}
```

Comment:
I don't think you need to check for if condition, because auth_posts will be empty list. But please double check it.